src/components/pages/Checkout/CheckoutProducts.js

```diff
@@ -11,8 +12,17 @@ const CheckoutProducts = ({ products, fetchProducts }) => {
     fetchProducts();
   }, [fetchProducts]);
 
```

Comment:
Maybe we should also use hooks here instead of mapStateToProps and mapDispatchToProps? It would look something like 
```
  const dispatch = useDispatch();
  const products = getProductsArray(useSelector(state => state.products.byId))

  useEffect(() => {
    dispatch(fetchProducts());
  }, [fetchProducts]);
```